docs/proxy/datacenter_proxy/examples.md

```diff
@@ -0,0 +1,668 @@
+    proxy_url = f"http://groups-SHADER+BUYPROXIES94952:{password}@proxy.apify.com:8000"
+    proxy_handler = request.ProxyHandler({
+        "http": proxy_url,
+        "https": proxy_url,
+    })
+
+    ctx = ssl.create_default_context()
+    ctx.check_hostname = False
+    ctx.verify_mode = ssl.CERT_NONE
+    httpHandler = request.HTTPSHandler(context=ctx)
+
+    opener = request.build_opener(httpHandler,proxy_handler)
+    return opener.open("https://api.apify.com/v2/browser-info").read()
+
+print(do_request())
+print("Should return a different clientIp than ")
+print(do_request())
+</marked-tab>
+
+
+<marked-tab header="Python 2+" lang="python">
+import six
+from six.moves.urllib import request
+import ssl
+
+def do_request():
+    # Replace <YOUR_PROXY_PASSWORD> below with your password
+    # found at https://my.apify.com/proxy
+    password = "<YOUR_PROXY_PASSWORD>"
+    proxy_url = (
+        "http://groups-SHADER+BUYPROXIES94952:%s@proxy.apify.com:8000" %
+        (password)
+    )
+
+    proxy_handler = request.ProxyHandler({
+        "http": proxy_url,
+        "https": proxy_url,
+    })
+
+    ctx = ssl.create_default_context()
+    ctx.check_hostname = False
+    ctx.verify_mode = ssl.CERT_NONE
+    httpHandler = request.HTTPSHandler(context=ctx)
+
+    opener = request.build_opener(httpHandler,proxy_handler)
+    return opener.open("https://api.apify.com/v2/browser-info").read()
+
+print(do_request())
+print("Should return a different clientIp than ")
+print(do_request())
+</marked-tab>
+
+
+<marked-tab header="PHP" lang="php">
+<?php
+function doRequest() {
+    $curl = curl_init("https://api.apify.com/v2/browser-info");
+    curl_setopt($curl, CURLOPT_RETURNTRANSFER, 1);
+    curl_setopt($curl, CURLOPT_PROXY, "http://proxy.apify.com:8000");
+    // Replace <YOUR_PROXY_PASSWORD> below with your password
+    // found at https://my.apify.com/proxy
+    curl_setopt($curl, CURLOPT_PROXYUSERPWD, "groups-SHADER+BUYPROXIES94952:<YOUR_PROXY_PASSWORD>");
+    $response = curl_exec($curl);
+    curl_close($curl);
+    return $response;
+}
+$response1 = doRequest();
+$response2 = doRequest();
+echo $response1;
+echo "\nShould have different clientIp than\n";
+echo $response2;
+?>
+</marked-tab>
+```
+
+## [](#using-the-apify-sdk) Using the Apify SDK
+
+If you're developing an actor using the [Apify SDK](https://sdk.apify.com), you can use Apify proxy in:
+
+* [PuppeteerCrawler](https://sdk.apify.com/docs/api/puppeteer-crawler#docsNav) using the [createProxyConfiguration()](https://sdk.apify.com/docs/api/apify#apifycreateproxyconfigurationproxyconfigurationoptions) function.
+* [requestAsBrowser()](https://sdk.apify.com/docs/api/utils#utilsrequestasbrowseroptions) function by specifying proxy configuration in the options.
+* [launchPuppeteer()](https://sdk.apify.com/docs/typedefs/launch-puppeteer#docsNav) by specifying the configuration in the function's options.
+
+### [](#single-random-ip-address) Single random IP address
```

Comment:
This will actually rotate IPs automatically, so the single IP might be a bit misleading. Yeah, it's a single IP for a single request, but that's always the case by definition.